app/components/UserList/UserListItem.tsx

```diff
@@ -9,120 +9,142 @@ interface UserListItemProps {
   username: string;
   isSelf: boolean | undefined;
   isDJ: boolean | undefined;
+  currentMiniRoom: string;
+  inConference: boolean;
 }
 
-const UserListItem = memo(
-  ({ userMachineId, username, isSelf, isDJ }: UserListItemProps) => {
-    const dispatch = useDispatch();
-    const audioRef = useRef();
-    const [isEditing, setIsEditing] = useState(false);
-    const [inputName, setInputName] = useState(username);
+const UserListItem = ({
+  userMachineId,
+  username,
+  isSelf,
+  isDJ,
+  currentMiniRoom,
+  inConference,
+}: UserListItemProps) => {
+  const dispatch = useDispatch();
+  const audioRef = useRef<HTMLAudioElement>();
+  const [isEditing, setIsEditing] = useState(false);
+  const [inputName, setInputName] = useState(username);
 
-    const conferenceUser = useSelector((state) => {
-      const { jitsiParticipants } = conferenceSelector(state);
-      const user = _.find(
-        jitsiParticipants,
-        ({ machineId }) => machineId === userMachineId
-      );
+  const conferenceUser = useSelector((state) => {
+    const { jitsiParticipants } = conferenceSelector(state);
+    return _.find(
+      jitsiParticipants,
+      ({ machineId }) => machineId === userMachineId
+    );
+  });
```

Comment:
`localConferenceUserSelector` from comment before could be reused here

```suggestion
  const conferenceUser = useSelector(localConferenceUserSelector(userMachineId));
```